tasks/internal/push-artifacts-to-cdn-task/push-artifacts-to-cdn-task.yaml

```diff
@@ -677,7 +682,8 @@ spec:
                 echo "$0: ERROR '$command' failed at line $line - exited with status $err" \
                   > "$(results.result.path)"
                 if [ -f "$STDERR_FILE" ] ; then
-                    tail -n 20 "$STDERR_FILE" >> "$(results.result.path)"
+                    # save output to the task result excluding the trace log
+                    tail -n 20 "$STDERR_FILE" |grep -v "^\+"| tee -a "$(results.result.path)"
```

Comment:
Actually, I am not sure this is good. This will first take the last 20 lines and then remove all lines starting with `+`. So if you're unlucky, you end up with zero lines. I think we should base this on characters, but also switch the order - first remove the debug lines and then do the tail. (This applies to all the instances of this.)

Reply:
order changed and also changed to bytes.

Reply:
I don't think it will work.
```
grep -v "^\+" |tail -c 512 "$STDERR_FILE" | tee -a "$(results.result.path)"
```
There is no input for the grep. You need to move the file argument.

Reply:
True. No idea why the tests passed as the grep should leave the prompt hanging.

Reply:
I think you forgot to say "fixed" ;) It seems it's fixed now, thanks.